telemetry/definitions/commonDefinitions.json

```diff
@@ -3645,6 +3645,48 @@
                 }
             ]
         },
+        {
+            "name": "cloudwatchlogs_startLiveTail",
+            "description": "When user starts a new LiveTail command",
+            "metadata": [
+                {
+                    "type": "filterType",
+                    "required": false
+                },
+                {
+                    "type": "hasTextFilter",
+                    "required": false
+                },
+                {
+                    "type": "result"
+                },
+                {
+                    "type": "sessionAlreadyStarted",
```

Comment:
Just FYI: if "sessionAlreadyStarted" is a kind of failure, it's strongly recommended to simply throw an exception and let `telemetry.xx.run(...)` automatically set `result=Failed` and `reason=<code>` where `<code>` is the error code.  https://github.com/aws/aws-toolkit-vscode/blob/016e478399f4e4e050e5e644c98c96297930e8ad/packages/core/src/shared/errors.ts#L144

That has other benefits: all failures can be captured in dashboards/queries easily, without needing to know specific, special fields such as this one.

However, if this is only an informational field and doesn't represent a failure or cancellation, then the above doesn't apply.

Reply:
We don't consider this to be a failure case. This would be an informational field. The `telemetry.xx.run(...)` will not raise an exception.